db/migrate/20230905160434_create_accessions.rb

```diff
@@ -0,0 +1,11 @@
+class CreateAccessions < ActiveRecord::Migration[7.0]
+  def change
+    create_table :accessions do |t|
+      t.string :druid, null: false
+      t.integer :version, null: false
+      t.timestamp :completed_at
```

Comment:
Since we are anticipating the need to track whether an accessioning fails #1241 should we add an `:errored_at` here, or wait till later?